pkg/analyze/redhat/clusteroperator.go

```diff
@@ -42,9 +47,100 @@ func (_ ClusterOperatorAnalyzer) Analyze(ctx context.Context, obj *status.Object
 		return status.UnknownStatusWithError(obj, err)
 	}
 
-	return analyze.AggregateResult(obj, nil, conditions)
+	relatedObjects, _, err := unstructured.NestedSlice(obj.Unstructured.Object, "status", "relatedObjects")
+	if err != nil {
+		// do not add any substatuses in case of error
+		return analyze.AggregateResult(obj, nil, conditions)
+	}
+
+	objectInfos := adaptRelatedObjects(obj, relatedObjects)
+
+	subStatuses := c.evaluateRelatedObjects(ctx, objectInfos)
+	return analyze.AggregateResult(obj, subStatuses, conditions)
+}
+
+func (c *ClusterOperatorAnalyzer) evaluateRelatedObjects(ctx context.Context, objectInfos []objectInfo) []status.ObjectStatus {
+	var statuses []status.ObjectStatus
+	for _, objInfo := range objectInfos {
+		if analyze.Register.IsIgnoredKind(c.evaluator.ResourceToKind(objInfo.groupResource).GroupKind()) {
+			continue
```

Comment:
Perhaps worth logging this with some high verbosity mode? Might be handly when trying to understand some caveats in the future.